<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigBase</class>
 <widget class="QDialog" name="ConfigBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuración base de datos</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Direcci&amp;ón del sevidor</string>
     </property>
     <property name="buddy">
      <cstring>lineEditDireccion</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditDireccion"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Puerto</string>
     </property>
     <property name="buddy">
      <cstring>lineEditPuerto</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditPuerto"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Usuario</string>
     </property>
     <property name="buddy">
      <cstring>lineEditUsuario</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditUsuario"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Contrase&amp;ña</string>
     </property>
     <property name="buddy">
      <cstring>lineEditClave</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditClave">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Base de datos</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditBaseDatos"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
